tests/find.spec.ts

```diff
@@ -93,18 +94,6 @@ describe('find', () => {
     expect(wrapper.find('.foo').exists()).toBe(true)
   })
 
-  it('returns the root element from dom wrapper if it matches', () => {
```

Comment:
This test now makes no sense, it captured previous behavior

I've added alternate test at the bottom of this file